### PR TITLE
rend now removes armor and vice versa

### PR DESCRIPTION
**Pull request by daedela**

### CODE REVIEW

File: src/act.offensive.cpp [hunk 1]
```diff
@@ -2881,18 +2882,39 @@ ACMD(do_rend) {
...
-        eff.location = APPLY_AC;
-        SET_FLAG(eff.flags, EFF_EXPOSED);
-        effect_to_char(vict, &eff);
+        if (affected_by_armor_spells(vict)) {
+            for (i = 0, counter = 0; armor_spell[i]; i++) {
+                if (affected_by_spell(vict, armor_spell[i])) {
+                    effect_from_char(vict, armor_spell[i]);
+                    act("&9&bYou rob $N of $S magical protection!&0", false, ch, 0, vict, TO_CHAR);
+                    act("&9&b$n robs $N of $S magical protection!&0", false, ch, 0, vict, TO_NOTVICT);
+                    act("&9&b$n robe you of your magical protection!&0", false, ch, 0, vict, TO_VICT);
```
Comment (stridera) [inline src/act.offensive.cpp:2902]: typoe:  Robs not robe

File: src/magic.cpp [hunk 2]
```diff
@@ -1141,6 +1141,13 @@ int mag_affect(int skill, CharData *ch, CharData *victim, int spellnum, int save
         if (check_armor_spells(ch, victim, spellnum))
             return CAST_RESULT_CHARGE;
 
+        if (affected_by_spell(victim, SKILL_REND)) {
```
Comment (stridera) [inline src/magic.cpp:1144]: Ugh, same thing with repeating here...

**Review by stridera — CHANGES_REQUESTED** [review]
Fix typo and then I'll approve.  :)

File: src/magic.cpp (hunk 2 above)
Comment (daedela) [reply src/magic.cpp:1144]: I don't understand this comment...?

**Review by stridera — APPROVED** [review]

**Pull request merged**